src/pages/Home.tsx

```diff
@@ -28,19 +37,23 @@ const Home: () => JSX.Element = () => {
             marginX={4}
             marginBottom={2}
             marginTop={['4', null, '0', null]}
-            height={['150px', null, '100%', null]}
+            height={['100%', null, '100%', null]}
```

Comment:
Note for mobile in the future: I think we should add the ability to "collapse" each box (including the map)

Reply:
I added an issue for this.